docs/.vitepress/vitepress/styles/css-vars.scss

```diff
@@ -55,7 +55,7 @@
   --content-min-width: 16rem;
   --content-max-width: 48rem;
 
-  --nav-z-index: 10;
+  --nav-z-index: 10000;
```

Comment:
If the sidebar is open, an exception might occur!

![image](https://user-images.githubusercontent.com/27342882/185390331-93cc3866-3127-4bde-9322-359663b96ca9.png)

Reply:
Thanks for your careful. There is also a problem when opening the dialog.

Reply:
Thinking about it, I thought that changing the z-index of .el-table-v2__overlay in the element-plus documentation might have less side effects. What do you think about it?

Changing the z-index of a component will affect the user.

![image](https://user-images.githubusercontent.com/27342882/185598945-79019b12-c87f-4a7d-89cc-5a8d40352fe5.png)

Reply:
You are right, totally agree.

Reply:
I have updated it, please review it again, thank you